eagleproject/core/blockchain/addresses.py

```diff
@@ -178,9 +176,11 @@
     LINK: "LINK",
     OGV: "OGV",
     OGV_BUYBACK: "OGV BuyBack",
-    REWARDS_SOURCE: "RewardsSource",
+    REWARDS_SOURCE: "RewardsSource"
 }
 
-# Also include metastrategies
-for strat in METASTRATEGIES:
-    CONTRACT_ADDR_TO_NAME[strat["ADDRESS"]] = strat["NAME"]
+# Also include strategies
+for (_, strat) in STRATEGIES.items():
```

Comment:
FYI, `STRATEGIES.values()`

Reply:
Ah, yes. Will update it

Reply:
eh, probably not worth fixing specifically, just making you aware if you weren't already.